dask_cuda/tests/test_spill.py

```diff
@@ -105,16 +108,20 @@ def test_device_spill(client, scheduler, worker):
         },
     ],
 )
-def test_cluster_device_spill(loop, params):
-    @gen_test()
-    def test_cluster_device_spill():
+def test_cluster_device_spill(params):
+    @gen_test(timeout=30)
+    def test():
+        loop = IOLoop.current()
+
         cluster = yield LocalCUDACluster(
             1,
+            loop=loop,
```

Comment:
I would expect this to use `IOLoop.current()` by default if asynchronous=True.  Is this not the case?

Reply:
You're right, we don't need it, at some point I was probably doing something wrong and it worked with this, I'm removing it now.